File: sdkwork-app-sdk-typescript/src/types/follow-check-vo.ts
/** 关注检查响应 */
export interface FollowCheckVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 是否已关注 */
  isFollowing?: boolean;
  /** 关注时间 */
  followTime?: string;
  /** 是否被对方关注 */
  isFollowedBy?: boolean;
}

File: sdkwork-app-sdk-csharp/Models/FollowCheckVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class FollowCheckVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public bool? IsFollowing { get; set; }
        public string? FollowTime { get; set; }
        public bool? IsFollowedBy { get; set; }
    }
}
